Makefile

```diff
@@ -98,6 +98,8 @@ scheduler: init
 sched_image: scheduler
 	@echo "building scheduler docker image"
 	@cp ${RELEASE_BIN_DIR}/${BINARY} ./deployments/image/configmap
+	@mkdir -p deployments/image/configmap/admission-controller-init-scripts
+	@cp ./deployments/admission-controllers/schedulername-mutation/*  deployments/image/configmap/admission-controller-init-scripts/
```

Comment:
We need to be consistent with the paths being prefaced with the `.` even when relative.
I think we get into problems when we don't do that. This is not just from this change but it has slipped in